openforcefield/topology/molecule.py

```diff
@@ -4697,6 +4700,137 @@ def add_conformer(self, coordinates):
 
         return self._add_conformer(coordinates)
 
+    def visualize(self, backend='rdkit', width=500, height=300):
+        """
+        Render a visualization of the molecule in Jupyter
+        
+        Parameters
+        ----------
+        backend : str, optional, default='rdkit'
+            Which visualization engine to use. Choose from:
+            - rdkit
+            - openeye
+            - nglview (conformers needed)
+        width : int, optional, default=500
+            Width of the generated representation (only applicable to
+            backend=openeye)
+        height : int, optional, default=500
```

Comment:
Ooops, this should be `300`

```suggestion
        height : int, optional, default=300
```